accessapproval/snippets/src/test/java/accessapproval/QuickstartIT.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package accessapproval;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.cloud.testing.junit4.StdOutCaptureRule;
+import com.google.common.base.Strings;
+import java.io.IOException;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class QuickstartIT {
+  private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+
+  @Rule public StdOutCaptureRule stdOutCap = new StdOutCaptureRule();
+
+  @BeforeClass
```

Comment:
Thanks for bringing the test file up for discussion earlier.
+1 to aligning the file according to the other tests in this repository.

Reply:
@suztomo Apologies if I wasn't clear in the earlier comment. Could you please update the test file to align with other examples in this repository?

Reply:
Ask

Reply:
Done:

- Renamed classes to reflect actions rather than QuickStart.
- Use assertWithMessage to assert environment variable
- Renamed `beforeAll()` to `setUp()`